src/main/java/com/budgetapp/entities/User.java

```diff
@@ -0,0 +1,39 @@
+package com.budgetapp.entities;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User extends BaseEntity{
+
+
+    private String username;
+    private String password;
+    private String roles;
+
+    public String getName() {
```

Comment:
Make sure your getters and setters are named correctly to the field.
```suggestion
    public String getUsername() {
```

Reply:
This still needs to be done